packages/components/src/utils/accessibility.ts

```diff
@@ -1,2 +1,16 @@
+import { StringOrTextWithA11y } from '../types/common'
+
 // Export related hooks
 export { useIsScreenReaderEnabled } from './hooks/useIsScreenReaderEnabled'
+
+/**
+ * Returns text that should be displayed on the screen
+ */
+export const getDisplayText = (text: StringOrTextWithA11y): string =>
+  typeof text === 'string' ? text : text.text
+
+/**
+ * Returns a11y override text or just text if a11y isn't provided
+ */
+export const getA11yText = (text: StringOrTextWithA11y): string =>
```

Comment:
Not a hook and exported:
```suggestion
export const GetA11yText = (text: StringOrTextWithA11y): string =>
```

Reply:
Also probably rename to GetA11yLabel corresponding to other comment on TextWithA11y type.

Reply:
Renamed function and use new a11yLabel field name